daemon/algod/api/server/lib/middlewares/auth.go

```diff
@@ -74,6 +74,10 @@ func Auth(log logging.Logger, apiToken string) func(http.Handler) http.Handler {
 			// Handle debug route
 			// Grab the apiToken from the HTTP header
 			providedToken := []byte(r.Header.Get(TokenHeader))
+			if len(providedToken) == 0 {
+				// Accept tokens provided in a bearer token format.
+				providedToken = []byte(r.Header.Get("Authorization")[7:])
```

Comment:
what does 7 mean?

Reply:
The pattern is `Authorization: Bearer <token>`, 7 just skips past `Bearer `

Reply:
Can we make this a little more explicit.  Get the "Authorization" header, look for "Bearer " (use a const bearerLabel), skip len(bearerLabel)

Reply:
I'd recommend using something like that. Currently also `frizzy YOUR-TOKEN` would work. 
`providedToken = strings.TeimPrefix(r.Header["Authorization"], "Bearer ")`

Reply:
Thanks for the suggestions, made the type checking a bit more explicit.